### DISCUSSION TITLE
wildy player warning (only warns if player lvl can atk you)

### DISCUSSION THREAD

**Discussion by runeliteissues** [runelite/runelite#17876] [Ideas]
instead of mass spamming ur screen everytime an acc shows up on screen, itd only activate if they can atk you

**Comment by Alexsuperfly** [comment]
Support for hub plugins should be directed to the author of the plugin. You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the ? button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the Support menu option.
